tests/test_forksafe.c

```diff
@@ -0,0 +1,103 @@
+/*********************************************************************
+  Blosc - Blocked Shuffling and Compression Library
+
+  Roundtrip compression/decompression tests.
```

Comment:
The summary is still wrong. I suggest "Tests that the library can be used after fork()"